lib/event.rb

```diff
@@ -0,0 +1,43 @@
+class Event
+  attr_accessor :name, :crafts, :attendees
+
+  def initialize(name, crafts, attendees)
+    @name = name
+    @crafts = crafts
+    @attendees = attendees
+  end
+
+  def attendee_names
+    names = @attendees.collect do |attendee|
```

Comment:
Nice use of `.collect` here. You do not need to create a variable though. It would still work without `names`.